vendor.mod

```diff
@@ -4,7 +4,9 @@
 
 module github.com/docker/docker
 
-go 1.21
+go 1.21.0
+
+toolchain go1.21.11
```

Comment:
should be removed

Reply:
Ah, yes, wanted to ask that; also curious if we could still keep the go1.21 (without .0), but maybe if doesn't allow that

Reply:
> if we could still keep the go1.21 (without .0),

nope: https://github.com/docker/buildkit-syft-scanner/pull/88#issuecomment-1859141638

Reply:
sigh. i was afraid of that

Reply:
Is there some flag that can be added to `vendor.sh` so that this isn't automatically added. I'm not sure what changes triggered it but obviously I did not manually add this.